lib/elixir/lib/enum.ex

```diff
@@ -2494,8 +2499,18 @@ defmodule Enum do
   defp enum_to_string(entry) when is_binary(entry), do: entry
   defp enum_to_string(entry), do: String.Chars.to_string(entry)
 
-  defp random_index(n) do
-    :rand.uniform(n + 1) - 1
+  defp random_integer(limit, limit) when is_integer(limit) do
+    limit
+  end
+
+  defp random_integer(lower_limit, upper_limit)
+      when is_integer(lower_limit) and is_integer(upper_limit) and upper_limit < lower_limit do
```

Comment:
It's possible to skip `is_integer(lower_limit) and is_integer(upper_limit)` check here since next clause won't match for non-integers anyway. :bowtie: